File: Common.UI/src/app/models/models/person.ts
import { Address } from './address';
import {
  IdentificationType,
  MaritalStatus,
  ContactInformation,
  Gender
} from '../index';

export class Person {
  firstName: string;
  lastName: string;
  dateOfBirth: Date;
  gender: Gender;
  identificationType: IdentificationType;
  identificationNumber: string;
  maritalStatus: MaritalStatus;
  nationality: string;
  email: string;

  address: Address = new Address();
  contactInformations: ContactInformation[] = [];
}

File: Common.Entities/Models/Person.cs
using Common.Entities.Enums;
using System;
using System.Collections.Generic;

namespace Common.Entities.Models
{
    public sealed class Person
    {
        public int PersonId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime DateOfBirth { get; set; }
        public Gender Gender { get; set; }
        public IdentificationType IdentificationType { get; set; }
        public string IdentificationNumber { get; set; }
        public MaritalStatus MaritalStatus { get; set; }
        public string Nationality { get; set; }
        public Address Address { get; set; }
        public string Email { get; set; }

        public ICollection<ContactInformation> ContactInformations { get; set; }
    }
}
